deep_qa/contrib/background_search/retrieval_word_overlap

```diff
@@ -0,0 +1,285 @@
+"""
```

Comment:
Can you rename this file to `retrieval_word_overlap.py`?  It'll make github show it correctly as a python file, and will make this a whole lot easier to read.